docs/_README.tpl.md

```diff
@@ -51,6 +51,8 @@ For an interactive demo of all components, see https://balena-io-modules.github.
 
 Wrap your application in the `<Provider>` component so that child components can correctly inherit the default theme. You can optionally provide your own theme.
 
+You can access theme object from child components using `useTheme` hook or `withTheme` higher-order component.
```

Comment:
Can you include an example here? Also, could you provide a heading with the title `Accessing the theme object`.
```suggestion
You can the access theme object from child components using the `useTheme` hook or the `withTheme` higher-order component.
```

Reply:
My grammar :))

There is an issue here. I can add `useTheme` example easily, but `withTheme` hoc is not exported from rendition, you have to import it from `styled-components`, so adding `withTheme` example would be wrong probably, until we export it from rendition.

Reply:
@karaxuna for now, lets remove the `withTheme` example until we have a better story for how to use it.